freckle-app/CHANGELOG.md

```diff
@@ -1,4 +1,17 @@
-## [_Unreleased_](https://github.com/freckle/freckle-app/compare/freckle-app-v1.20.3.0...main)
+## [_Unreleased_](https://github.com/freckle/freckle-app/compare/freckle-app-v1.21.0.0...main)
+
+## [v1.21.0.0](https://github.com/freckle/freckle-app/compare/freckle-app-vv1.20.3.0...freckle-app-v1.21.0.0)
+
+The following have moved from `Freckle.App.Database` to a new package, `persistent-sql-lifted`:
+
+- `MonadSqlTx (..)`
+- `HasSqlBackend (..)`
+- `MonadSqlBackend (..)`
+- `liftSql`
+
+The definitions in `freckle-app` have been replaced with reëxports,
```

Comment:
> reëxports

curious how you pronöunce this.

Reply:
[Diaeresis](https://en.wikipedia.org/wiki/Diaeresis_(diacritic))

> used to indicate the separation of two distinct vowel letters in adjacent syllables

e.g. in *naïve*